lib/src/visitor/async_evaluate.dart

```diff
@@ -42,7 +42,8 @@ import 'interface/modifiable_css.dart';
 import 'interface/statement.dart';
 
 /// A function that takes a callback with no arguments.
-typedef _ScopeCallback = Future Function(Future Function() callback);
+typedef _ScopeCallback = Future<void> Function(
```

Comment:
I really like these fixes; nice for readers to see that something is  `Future<void>`.